src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -812,6 +812,13 @@ private bool Synchronize()
             lock (this.lockObj)
             {
                 HashHeightPair tipToChase = this.TipToChase();
+                if (this.TipHashAndHeight == null)
+                {
+                    this.logger.LogError("Failed to synchronise. Reason: TipHashAndHeight is null.");
```

Comment:
Change to `this.logger.LogError("Failed to synchronise. Reason: {0} is null.", nameof(TipHashAndHeight));`